std/pdb.zig

```diff
@@ -558,7 +558,7 @@ const SuperBlock = packed struct {
     BlockSize: u32,
 
     /// The index of a block within the file, at which begins a bitfield representing
-    /// the set of all blocks within the file which are “free” (i.e. the data within
```

Comment:
these changes are an improvement independent of this PR. you could make these a separate PR.

Reply:
This PR doesn't pass without fixing all the broken code. This code had non-UTF8.